aristanetworks/ansible-avd
# Feat: AVD dev containers

## Change Summary

First step in implementing AVD dev containers support
Tracked in #3178 

## Component(s) name

Github actions and packages

## Proposed changes
Adding first workflows and dev container definitions

## How to test
A [dedicated repository](https://github.com/ankudinov/ansible-avd-experimental-copy) will be used for initial testing.
Once merged, we must verify successful container builds and container functionality.
